### PR TITLE
[10.0][ADD] l10n_ch_account_reconcile_isr

### PR DESCRIPTION
**Pull request by tschanzt**

### CODE REVIEW

File: l10n_ch_account_reconcile_esr/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Switzerland - ESR account reconcile',
+    'version': '10.0.1.0.0',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'website': 'http://www.camptocamp.com',
```
Comment (guewen) [inline l10n_ch_account_reconcile_esr/__manifest__.py]:
```suggestion
    'website': 'https://github.com/OCA/l10n-switzerland',
```

File: l10n_ch_account_reconcile_esr/models/account_bank_statement_line.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        else:
+            return self.auto_reconcile_esr()
+
+    @api.multi
+    def auto_reconcile_esr(self):
+        """ Reconcile the Bank statement line and acount move line
+        based on only the esr
+
+        """
+        # If we don't have esr_reconcile we want the default odoo way
```
Comment (guewen) [inline l10n_ch_account_reconcile_esr/models/account_bank_statement_line.py]: This comment was meant for the `auto_reconcile` method I guess

**Review by guewen — APPROVED** [review]
A couple of nitpicks, but LGTM

**Comment by tschanzt** [comment]
@guewen updated.

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@tschanzt :red_circle:

File: l10n_ch_account_reconcile_esr/i18n/it.po [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+#, python-format
+msgid "ESR reconciliation"
+msgstr "Riconciliazione PVR"
+
+#. module: l10n_ch_account_reconcile_esr
+#. openerp-web
+#: code:addons/l10n_ch_account_reconcile_esr/static/src/xml/account_reconciliation.xml:9
+#, python-format
+msgid "Let odoo try to reconcile entries for the user using esr refs"
+msgstr ""
```
Comment (simahawk) [inline l10n_ch_account_reconcile_esr/i18n/it.po]: @eLBati may I ask your help here? :pray: 
Does this sound good to you? "Consenti a Odoo di riconciliare le linee usando i riferimenti ESR"

Comment (eLBati) [reply l10n_ch_account_reconcile_esr/i18n/it.po]: Io metterei "Consenti a odoo di provare a riconciliare le registrazioni usando i riferimenti ESR"

Comment (simahawk) [reply l10n_ch_account_reconcile_esr/i18n/it.po]: @tschanzt pick this :wink: 
@eLBati tnx!

File: l10n_ch_account_reconcile_esr/models/account_bank_statement_line.py [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+
+class AccountBankStatementLine(models.Model):
+    _inherit = "account.bank.statement.line"
+
+    @api.multi
+    def auto_reconcile(self):
+        # If we don't have esr_reconcile we want the default odoo way
+        if not self.env.context.get('esr_reconcile'):
+            return super(AccountBankStatementLine, self). auto_reconcile()
```
Comment (leemannd) [inline l10n_ch_account_reconcile_esr/models/account_bank_statement_line.py]:
```suggestion
            return super(AccountBankStatementLine, self).auto_reconcile()
```

**Review by leemannd — CHANGES_REQUESTED** [review]
One error spotted

**Review by leemannd — APPROVED** [review]
Thanks

File: l10n_ch_account_reconcile_esr/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Switzerland - ESR account reconcile',
```
Comment (yvaucher) [inline l10n_ch_account_reconcile_esr/__manifest__.py]: Please use the english name "ISR".

We did it wrong in the past by using BVR everywhere. But the official term in english is ISR for In-payment Slip with Reference.

**Review by yvaucher — CHANGES_REQUESTED** [review]

**Comment by tschanzt** [comment]
@yvaucher renamed & replaced all occurences

**Review by yvaucher — APPROVED** [review]

**Pull request merged**